DOCUMENTATION.md

```diff
@@ -107,19 +122,162 @@ In case it's a **dictionary** with fields, please make sure the **encoder** func
 - `confirmation`: **boolean**  this boolean indicates if you wish to receive a confirmation after sending the downlink message. Default value to False.
 - `schedule`: **string**  this string provides the type of schedule on which the message should be sent. It can take values such as `first` or `last`. Default value to `replace`.
```

Comment:
What are the different values that can be given? If it's too long, provide a link